### PR TITLE
Airbrake integration improvements

### PR DESCRIPTION
**Pull request by lucianopanaro**
- Update Airbrake dependency to 'airbrake' instead of 'airbrake-ruby' (See https://github.com/airbrake/airbrake)
    - Adds Rack middleware: it relies on the exception being stored in a Rack env var (set in a `rescue_from :all` call)
    - Had to rename config/airbrake.rb to config/airbrake_load.rb to avoid a name collision with the airbrake gem
    - Adds airbrake for Sidekiq (run bin/sidekiq -v to see that the
      middleware is added)

### CODE REVIEW

File: app/api/v1/defaults.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ module Defaults
 
         # Global handler for any unexpected exception
         rescue_from :all do |e|
-          Airbrake.notify(e)
+          env['rack.exception'] = e # Store it in Rack env for Airbrake middleware
```
Comment (excelsior) [inline app/api/v1/defaults.rb:39]: Question: the `Airbrake.notify` method is used in other places, too. In one case it sends a payload alongside the exception, in another it sends an error message without the actual exception. How are those two cases supposed to work now? Or is it OK to continue using `Airbrake.notify` still?

**Review by excelsior — COMMENTED** [review]
Thanks @lucianopanaro, looks good. Could you address the question I have about the new approach please?

File: app/api/v1/defaults.rb (hunk 1 above)
Comment (lucianopanaro) [reply app/api/v1/defaults.rb:39]: `Airbrake.notify` can still be called in any scenario, no problem or conflict at all.

If you want to include request metadata in the notification you can use Rails' `notify_airbrake` helper as reference: https://github.com/airbrake/airbrake/blob/68bec4972aa3bbc75014f8f67ea7ea5213b0a02c/lib/airbrake/rails/controller_methods.rb#L31

**Comment by lucianopanaro** [comment]
Answered @excelsior, ty!

**Pull request merged**